modules/images/image-loading-optimization/class-ilo-url-metrics-group-status.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Image Loading Optimization: ILO_URL_Metrics_Group_Status class
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Status of a given URL metric group.
+ *
+ * @phpstan-type Data array{
+ *                        minimumViewportWidth: int,
+ *                        isLacking: bool,
+ *                    }
+ *
+ * @since n.e.x.t
+ * @access private
+ */
+final class ILO_URL_Metrics_Group_Status implements JsonSerializable {
```

Comment:
See above, I think this class is too specific. I don't see the need for it, but if this information needs to be in a dedicated class per group, I think it would make more sense in a general "URL metric group" class.